.github/CODEOWNERS

```diff
@@ -1,2 +1,3 @@
+* @brave/devops-team @brave/uplift-approvers
```

Comment:
This line might have to be last.

Reply:
In the example at https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file, it's the first line.

Looks like uplift-approvers needs to have write access to the repo to be correctly set as code owners. They currently have read. Will bump to write, seems fine.

Reply:
Yep, that fixed it.